CHANGELOG.md

```diff
@@ -4,7 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v11.0.0](https://github.com/voxpupuli/puppet-rabbitmq/tree/v11.0.0) (2021-01-16)
+## [Unreleased](https://github.com/voxpupuli/puppet-rabbitmq/tree/HEAD)
```

Comment:
this looks odd, it should be 11.1.0 and not unreleased. did you generate the changelog before updating metadata.jdon?

Reply:
Derp. You’re right. I’ll regenerate